scripts/engines/jawsm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+git clone https://github.com/drogus/jawsm.git --depth 1
+
+cd jawsm
+git rev-parse HEAD >../version.txt
+
+cargo build --release
```

Comment:
Cargo can't handle "many cores, limited RAM", this risks OOMing the runner.

https://github.com/test262-fyi/test262.fyi/blob/753db7768f9d9e760b91361d2d399cd8b97a0677/scripts/engines/nova.sh#L9